src/wazuh_modules/vulnerability_scanner/src/scanOrchestrator/packageScanner.hpp

```diff
@@ -201,26 +200,10 @@ class TPackageScanner final : public AbstractHandler<std::shared_ptr<TScanContex
      */
     std::string getCNA(std::shared_ptr<TScanContext> ctx)
     {
-        auto cnaName {m_databaseFeedManager->getCnaNameByFormat(ctx->packageFormat().data())};
-
-        if (cnaName.empty())
-        {
-            cnaName = m_databaseFeedManager->getCnaNameBySource(ctx->packageSource().data());
-            if (cnaName.empty())
-            {
-                cnaName =
-                    m_databaseFeedManager->getCnaNameByPrefix(ctx->packageVendor().data(), ctx->osPlatform().data());
-                if (cnaName.empty())
-                {
-                    cnaName = m_databaseFeedManager->getCnaNameByContains(ctx->packageVendor().data(),
-                                                                          ctx->osPlatform().data());
-                    if (cnaName.empty())
-                    {
-                        return DEFAULT_CNA;
-                    }
-                }
-            }
-        }
+        auto cnaName {m_databaseFeedManager->getCNA(ctx->packageFormat().data(),
+                                                    ctx->packageSource().data(),
+                                                    ctx->packageVendor().data(),
+                                                    ctx->osPlatform().data())};
```

Comment:
We need to save this cna value and the expanded cna value in a tuple in the scanContext

```
cna -> suse
expandedCna -> suse_desktop_15

scan context
    m_vulnerabilitySource <-- std::tuple<string,string>(suse, suse_desktop_15)
```

@pereyra-m Can use this field as well for the descriptions

Reply:
+1: Using the scan context reduces searching the same value many times